packages/utils/src/file.ts

```diff
@@ -1,5 +1,10 @@
 const units = ['B', 'KB', 'MB', 'GB', 'TB'];
 
+/**
+ *
+ * @param bytes amount fo bytes
+ * @returns String with the human redable size of bytes
+ */
 export const toHumanReadable = (bytes: number): string => {
```

Comment:
Better name is always better that comment, what about `bytesToHumanReadable`?

Reply:
:+1: